data/experiment-recipe-samples/desktop-97-rollout.json

```diff
@@ -0,0 +1,48 @@
+{
+  "appId": "firefox-desktop",
+  "appName": "firefox_desktop",
+  "application": "firefox-desktop",
+  "arguments": {},
+  "isRollout": true,
+  "branches": [
+    {
+      "feature": {
+        "enabled": true,
+        "featureId": "urlbar",
+        "value": {
+          "firefoxSuggestLabelsEnabled": true,
+          "quickSuggestEnabled": false,
+          "quickSuggestNonSponsoredIndex": -1,
+          "quickSuggestShouldShowOnboardingDialog": true,
+          "quickSuggestShowOnboardingDialogAfterNRestarts": 2,
+          "quickSuggestSponsoredIndex": -1
+        }
+      },
+      "ratio": 1,
+      "slug": "control"
+    }
+  ],
+  "bucketConfig": {
+    "count": 1700,
+    "namespace": "urlbar-21-rollout",
```

Comment:
Added `-rollout` to the namespace, to simulate [the nimbus model logic here](https://github.com/mozilla/experimenter/blob/main/app/experimenter/experiments/models/nimbus.py#L422-L423)